test/unit/property-effects.html

```diff
@@ -1760,6 +1760,51 @@
 
 });
 
+suite('ES6 getters/setters', function() {
+  test('notify path bypasses internal cache to update template', function(done) {
+    let el = document.createElement('x-getters-setters');
+    el.prop1 = 'prop1';
+    document.body.appendChild(el);
+    setTimeout(() => {
+      let originalProp2 = el.$.prop2.innerText;
+      assert.equal(el.$.prop1.innerText, 'prop1');
+      assert.equal(originalProp2.indexOf('prop2'), 0);
+      assert.equal(el.$.addProps.innerText.indexOf('prop1 prop2'), 0);
+      assert.equal(el.$.prop1TwoWay.value, 'prop1');
+      let originalProp2TwoWay = el.$.prop2TwoWay.value;
+      assert.equal(originalProp2TwoWay.indexOf('prop2'), 0);
+      
+      el.prop1_ = 'prop1changed';
```

Comment:
`prop2` has a getter but no setter. It's read only. Do you want me to add a 3rd property to test that?

I initially had that but removed it. Since `PropertyEffects` overrides `PropertyAccessor` to make `_propertiesChanged` synchronous, I removed the 3rd property.